src/components/language-toggle/README.md

```diff
@@ -110,8 +110,16 @@ The package is owned by the DI Frontend Capability team, part of the development
      ])
    );
    ```
+6. Add a middleware function to pass the current URL as a parameter to the configuration of the Nunjucks environment
```

Comment:
In this update of the package readme, we need to add a paragraph about the nunjuck filter.